scripts/ROPgadget

```diff
@@ -1,12 +0,0 @@
-#!/usr/bin/env python
-## -*- coding: utf-8 -*-
-##
-##  Jonathan Salwan - 2014-05-12 - ROPgadget tool
-##
-##  http://twitter.com/JonathanSalwan
-##  http://shell-storm.org/project/ROPgadget/
-##
-
-import ropgadget
-
-ropgadget.main()
```

Comment:
why do you remove this script?

Reply:
For compliance and modern python3 project structure. The goal is to fit the PEP 621 by having a cleaner structure with `pyproject.toml` that will fit all the needs 😄! It allows us to use tools such as `pipx` or just even `pip install .` inside the project and have a perfect `ropgadget` in our environment.